sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_receiver_async.py

```diff
@@ -78,6 +78,12 @@ async def _incoming_transfer(self, frame):
                 self._received_payload = bytearray()
             else:
                 message = decode_payload(frame[11])
+            _LOGGER.debug(
+                "Received message: annotations: %r, header: %r",
+                message.message_annotations,
+                message.header,
```

Comment:
I don't think adding header here should expose any private info. I think it's useful to see delivery_count/ttl that may  exist in the header in some cases. Thoughts/concerns? Are there any scenarios I'm missing where this does expose info that we don't want to?

Reply:
I don't think so, one thing if we are exposing the locked_until_time, could we also log it when it gets decoded into utc time so its more human legible?

Reply:
sounds good . OOB: do if logging_enable, output only values where key is x-opt- (i.e. service vals, so we don't leak customer props) + rest.

Reply:
if logging.getLogger().isEnabledFor(logging.DEBUG):

Reply:
updated!